packages/create-sitecore-jss/watch.json.example

```diff
@@ -12,9 +12,12 @@
     "language": "da-DK",
     "fetchWith": "GraphQL",
     "prerender": "SSG",
+    // when using the "nextjs-xmcloud" add-on, set "xmcloud" to true
+    "xmcloud": false,
```

Comment:
It's better to ensure 'xmcloud' is being set when xmcloud initializer is invoked on `create-sitecore-jss` side, rather than rely on xmcloud being explicitly set in yarn.watch _and_ `npx ...` command.
But if ComponentLibrary component is added inline, perhaps we won't need this at all.

Reply:
`xmcloud` is already set to true in `create-sitecore-jss`, this change is just to align with local development since watch script relies on `args` from the` watch.json` file.